File: ADS-2-Semestre/EngenhariaDeSoftware/InterfacePatoC#/Pato/PatoBorracha.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace JogoPato
{
	public class PatoBorracha : Pato
    {
		public PatoBorracha()
		{
			setComportamento(new NaoVoa());
		}

		public override string mostrar()
		{
			return "Olá, eu sou de Boarracha.";
		}
	}
}

File: ADS-2-Semestre/EngenhariaDeSoftware/InterfacePatoCorredor/Pato_Borracha.java
package engsoft;

public class Pato_Borracha extends Pato {

	public Pato_Borracha()
	{
		setComportamento(new Nao_Voa());		
	}
	
	public String mostrar() {
		return "Olá, eu sou de Boarracha.";
	}
	
}
